<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_3</class>
 <widget class="QWidget" name="page_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1150</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="select_Edit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>610</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>工号/姓名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_Btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>610</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_Btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>610</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
